frontend/src/lib/k8s/api/v2/fetch.ts

```diff
@@ -0,0 +1,84 @@
+import helpers from '../../../../helpers';
+import { findKubeconfigByClusterName, getUserIdFromLocalStorage } from '../../../../stateless';
+import { getToken, setToken } from '../../../auth';
+import { getClusterAuthType } from '../v1/clusterRequests';
+import { refreshToken } from '../v1/tokenApi';
+import { makeUrl } from './makeUrl';
+
+export const BASE_HTTP_URL = helpers.getAppUrl();
+
+/**
+ * Simple wrapper around Fetch function
+ * Sends a request to the backend
+ *
+ * @param url - URL path
+ * @param init - same as second parameter of the Fetch function
```

Comment:
```suggestion
 * @param init - options parameter for the Fetch function
```
I think this could be a little more descriptive, just a nit though. Feel free to ignore